src/warning/warning-printer.ts

```diff
@@ -52,6 +52,13 @@ export class WarningPrinter {
   async printWarning(warning: Warning) {
     const severity = this._severityToString(warning.severity);
     const range = warning.sourceRange;
+    if (!range) {
+      this._outStream.write(
```

Comment:
I understand not wanting to bother the user too much, but I think both they and us would benefit from having the message here as well. It could include important additional info, and can always be remove after we've found the specific error you're chasing down atm.